data/puppy_seed.json

```diff
@@ -0,0 +1,99 @@
+	{"src": "http://36.media.tumblr.com/afb22471e3f234eb0027e95fa179270c/tumblr_o322owt60b1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://40.media.tumblr.com/252a41b65f2327de5ac48208b925643c/tumblr_o322gkqybI1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://41.media.tumblr.com/8fa299bac909e1798d79f631252dbe3c/tumblr_o322nzBz7H1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://40.media.tumblr.com/4e8bcd0c73e8c07fb95264f9487c89bf/tumblr_o322id0U6c1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://41.media.tumblr.com/f5cb62922bf86b8d77ed870d1272694c/tumblr_o3010ofrNL1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://40.media.tumblr.com/e8431529486f278b805e65cf3a80819c/tumblr_o300hm2BOM1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://41.media.tumblr.com/5e81782c09160526e069ea1ba65e329b/tumblr_o3001sHMIy1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://41.media.tumblr.com/e8ed5308c1f02624d98fc52451c9af46/tumblr_o2zztj9e2Q1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://36.media.tumblr.com/9de331f903e853915373a28d18636f51/tumblr_o2y9fnK5be1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://40.media.tumblr.com/f7459cb09c3ceef17699a064ef21b5bc/tumblr_o2y94z3laj1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://40.media.tumblr.com/f115bc2d3523d3b576a986f9f8e9487b/tumblr_o2y7jvrOam1ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://40.media.tumblr.com/4cdcfd8ce7e76b79b215c32b0510fb3c/tumblr_o2y7g8pKn31ujzn7ko1_500.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://s-media-cache-ak0.pinimg.com/564x/59/be/1e/59be1e6b950d67540412177131f63f31.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://s-media-cache-ak0.pinimg.com/564x/c1/e2/12/c1e212fb5552d947b6f07b9470f793d7.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://s-media-cache-ak0.pinimg.com/564x/04/e4/fb/04e4fb941625f694b12e79881cff91ba.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://wesleying.org/wp-content/uploads/2015/11/cute-dogs-01.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "https://mbtimetraveler.files.wordpress.com/2016/01/cute-dog-names-11.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0},
+	{"src": "http://s3.amazonaws.com/assets.prod.vetstreet.com/64/5c/2914709f4bc38fcca4126dfa2c74/american-eskimo-dog-AP-GE1BT2-590sm52213.jpg", "losses": 0, "wins": 0, "rating": 1200, "reportCount": 0}
+]
```

Comment:
Dude, where is the new line.

#UseVIM

#SquashThis

#YouHaveToBecauseDarrenAlwaysMadeMeDoIt

Also it's ugly and if I open that file Vim will auto add a new line